content/numpy/concepts/built-in-functions/terms/dot/dot.md

```diff
@@ -0,0 +1,84 @@
+---
+Title: '.dot()'
+
+Description: 'Dot product of two arrays.' 
+
+Subjects: 
+  - 'Data Science'
+  - 'Computer Science'
+
+Tags: 
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Linear Algebra'
+  - 'Functions'
+  - 'NumPy'
+
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'  
+---
+
+The **`np.dot()`** function computes the dot product of two arrays `a` and `b`, with the result varying based on the input. The function returns an [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray), and if both `a` and `b` are either scalars or 1-D arrays, a scalar is returned. Otherwise, an array is returned.
+
+## Syntax
+
+```pseudo
+numpy.dot(a, b, out=None)
+```
+Parameters:
+- 'a: array_like, first argument'
+- 'b: array_like, second argument'
+- 'out: ndarray, optional'
+
+Returns:
+- output:ndarray 
+
+## Example
+
+```py
+import numpy as np
+
+# defining 2 arrays
+array_a = [3,4,5]
+array_b = [1,2,3]
+
+# computing and printing the product
+print(np.dot(array_a, array_b))
+```
+```shell
+26
+```
+
+## Codebyte Example
+```py
+import numpy as np
+
+# creating two matrices 
+m = [[3, 5], [1, 4], [5, 4]] 
+n = [[7, 3, 1], [2, 7, 2]] 
+print("Matrix m :") 
+print(m) 
+print("Matrix n :") 
+print(n) 
+  
+# computing product 
+result = np.dot(m, n) 
+  
+# printing the result 
```

Comment:
```suggestion
```

Reply:
Please accept the above code byte suggestions, since comments are not required for a codebyte.